pkgs/by-name/ta/tatl/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  stdenv,
+  lib,
+  fetchzip,
+}:
+
+stdenv.mkDerivation {
+  name = "TATL*";
+  version = "20241001"; # They have no version number so using the time of writing this package as version.
+  src = fetchzip {
+    url = "https://atila.ibisc.univ-evry.fr/tableau_ATL_star/bin/tatl_star.tar.gz";
```

Comment:
If they update the package without changing the URL, you need to use an archived copy. Unfortunately the wayback machine currently doesn't accept new archives.

Reply:
It's also questionable whether an archive URL for a nonfree package would even be legal, given that you would be distributing the package.

Reply:
I suppose we could ask the developers whether they would be willing to version their archives.

Reply:
Ok. I'll see if I can track down the developers, ask them, and possibly also license the tool while they're at it.

Reply:
Did you get a response?

Reply:
More or less. I'm in contact with the author and we are talking about open sourcing TATL which would make this whole package much nicer.